addons/api/.eslintrc.js

```diff
@@ -2,13 +2,17 @@
 
 module.exports = {
   root: true,
-  parser: 'babel-eslint',
+  parser: '@babel/eslint-parser',
   parserOptions: {
     ecmaVersion: 2018,
     sourceType: 'module',
     ecmaFeatures: {
```

Comment:
@randallmorey 
I think this is legacy from the parser we are deprecating. After we merge this, I would like to perform some testing to triple check before deleting it.

Reply:
Never mind my previous comment. As per discussion on FE tech time through zoom we will try to deprecate this old code within this PR.